subtrack.MAUI/Pages/Index.razor

```diff
@@ -50,12 +50,19 @@
     protected override async void OnInitialized()
     {
         var subscriptions = await subscriptionService.GetAllAsync();
-        _subscriptionsCost = subscriptionsCalculator.GetTotalCost(subscriptions);
+        _subscriptionsCost = subscriptionsCalculator.GetAverageMonthlyCost(subscriptions);
         _subscriptionResponses = SubscriptionMapper.ToResponses(subscriptions, subscriptionsCalculator)
                                                    .OrderBy(sub => sub.DueDays)
                                                    .ToList();
     }
 
+    private string FormatAmount()
+    {
+        var rounded = Math.Round(_subscriptionsCost, 2);
+        var prefix = _subscriptionsCost == rounded ? "" : "~";
```

Comment:
Its always going to have "~" since its no longer the actual cost of the month but average cost. So you can directly when getting cost assign the field with the math.round and in the html just ~Cost